### PR TITLE
add NetworkStats struct

### PR DESCRIPTION
**Pull request by donhcd**
inline structs are pretty terrible, especially for people who want to use those structs in their own data structures

### CODE REVIEW

**Comment by donhcd** [comment]
ping @vieux @aluzzardi

File: types.go [hunk 1]
```diff
@@ -237,19 +248,9 @@ type BlkioStats struct {
...
-		TxPackets uint64 `json:"tx_packets"`
-		TxErrors  uint64 `json:"tx_errors"`
-		TxDropped uint64 `json:"tx_dropped"`
-	}
-
-	CpuStats    CpuStats    `json:"cpu_stats,omitempty"`
-	MemoryStats MemoryStats `json:"memory_stats,omitempty"`
-	BlkioStats  BlkioStats  `json:"blkio_stats,omitempty"`
+	Read         time.Time    `json:"read"`
+	NetworkStats NetworkStats `json:"network_stats,omitempty"`
```
Comment (vieux) [inline types.go]: Shouldn't this be just `network` ?

**Comment by vieux** [comment]
ping @crosbymichael

File: types.go (hunk 1 above)
Comment (donhcd) [inline types.go]: yes, good catch

**Comment by donhcd** [comment]
@vieux updated "network_settings" -> "network"

**Comment by crosbymichael** [comment]
@vieux you all depend on this more than what I do, feel free to review without me

**Comment by ehazlett** [comment]
LGTM

**Comment by vieux** [comment]
LGTM

**Pull request merged**